src/components/FeedPanel/FeedItem.scss

```diff
@@ -36,29 +36,65 @@
 
 
 .feed-item {
-	border: 1px solid $darkBG;
-	box-sizing: border-box;
-	border-radius: 15px;
-	padding: 10px;
-	width: 100%;
-	margin: 5px 0px;
-	margin-bottom: 5px;
+    background-color: hsl($basehue,$basesaturation, percentage(($baselighgting * 3) / 255));
```

Comment:
We need to use a theme for this. so either set add it in the theme or remove it.